src/components/tech/WorkingBox.tsx

```diff
@@ -0,0 +1,75 @@
+import {
+  Badge,
+  Box,
+  Container,
+  Flex,
+  Heading,
+  Link,
+  Section,
+  Text,
+} from "@radix-ui/themes";
+import React from "react";
+
+interface WorkingBoxProps {
+  title: string;
+  para: string;
+  link: string;
+  tech: string;
+}
+
+const colors: { [key: number]: string } = {
+  0: "#3E63DD",
+  1: "#5AC597",
+  2: "#98BEC6",
+  3: "#DFCDE8",
+  4: "#4362A6",
+};
+
+export const WorkingBox = ({ title, para, link, tech }: WorkingBoxProps) => {
+  const data = tech.split(",");
+  return (
+    <Flex
+      gap="4"
+      align="start"
+      direction="column"
+      p="20px"
+      width="full"
+      justify="start"
+      maxWidth={{ md: "610px", initial: "auto" }}
```

Comment:
**question:** Would it be more maintainable to wrap this `<Flex>` in a [`<Container>`](https://www.radix-ui.com/themes/docs/components/container) to set the max width?  Size 2 containers have a max width of 688px.

Reply:
**suggestion (non-blocking):** Always put the `initial` attribute first, so that we can understand the sizes as they grow larger from left to right.  Please review other changes in this PR to ensure they also follow this code convention. :)